### PR TITLE
Edited Parallel Simulations

### PR DESCRIPTION
**Pull request by maiyahershey**
This PR addresses the How to: run simulations in parallel: 
1- performed edits on the text
2- split the information into smaller chunks or steps to follow the "how-to guide style"

### CODE REVIEW

File: docs/how_to/parallel_simulations.md [hunk 1]
```diff
@@ -1,61 +1,69 @@
...
 
-Machine groups can be used to run multiple simulations in 
-parallel, by having each of the machines in the group run a separate simulation.
-This is a powerful mechanism to reduce waiting times when you need to run a large
-number of simulations, such as when you are exploring different parameter values
-or running a large number of simulations for a sensitivity analysis. 
+Running multiple simulations in parallel can significantly reduce waiting times, 
+especially useful when exploring various parameter values or running a large number 
+of simulations for a sensitivity analysis. This how-to guide will walk you through 
+using Machine Groups to run several simulations in parallel, using the [templating mechanism](../explore_api/templating.md) integrated within the Inductiva API. This approach makes
```
Comment (sergiomsantos) [inline docs/how_to/parallel_simulations.md:6]:
```suggestion
using Machine Groups to run several simulations in parallel, using the
[templating mechanism](../explore_api/templating.md) integrated within the
Inductiva API. This approach makes 
```

File: docs/how_to/parallel_simulations.md [hunk 2]
```diff
@@ -1,61 +1,69 @@
...
 
+## 4. Monitoring Simulations
+
 The template mechanism will allow you to explore 5 different variations of the
 simulation, each with a different water level. The simulations will be submitted
 to our dedicated machine group and will run in parallel.
-We can check that all simulations are running via the CLI and that it took only
-1min for the moment they are submitted until they start running:
+
+You can check the status of these simulations through the [Inductiva CLI](../cli/streaming-logs.md), and you'll see that it took only **1 minute** from the moment they were submitted until they start running:
```
Comment (sergiomsantos) [inline docs/how_to/parallel_simulations.md:66]:
```suggestion
You can check the status of these simulations through the
[Inductiva CLI](../cli/streaming-logs.md), and you'll see that it took only **1 minute**
from the moment they were submitted until they start running:
```

**Review by sergiomsantos — APPROVED** [review]
It's becoming hard to map the `*md` source files to the rendered structure.
For example, this file lives in `how_to` and is called `parallel_simulations`. Yet, it is rendered in `Guides/Run Multiple Simulations in Parallel`. I believe we could map the source structure to mirror the rendered one. Wdyt?

**Comment by maiyahershey** [comment]
> It's becoming hard to map the `*md` source files to the rendered structure. For example, this file lives in `how_to` and is called `parallel_simulations`. Yet, it is rendered in `Guides/Run Multiple Simulations in Parallel`. I believe we could map the source structure to mirror the rendered one. Wdyt?

Yes agreed. let me do that

**Pull request merged**